addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -71,8 +71,8 @@
                     <field name="tax_ids" optional="hide" widget="many2many_tags" groups="account.group_account_invoice,account.group_account_readonly"/>
                     <field name="amount_tax_company" optional="hide" groups="account.group_account_invoice,account.group_account_readonly"/>
                     <field name="attachment_number" nolabel="1" class='fa fa-paperclip pe-0' attrs="{'invisible': [('attachment_number', '=', 0)]}"/>
-                    <field name="total_amount_company" optional="show" sum="Total Company Currency" widget='monetary' options="{'currency_field': 'company_currency_id'}" decoration-bf="True"/>
-                    <field name="total_amount" optional="hide" decoration-bf="True" groups="base.group_multi_currency"/>
+                    <field name="total_amount_company" sum="Total Company Currency" widget='monetary' options="{'currency_field': 'company_currency_id'}" decoration-bf="1"/>
+                    <field name="total_amount" widget='monetary' options="{'currency_field': 'currency_id'}" optional="hide" decoration-bf="1" groups="base.group_multi_currency"/>
```

Comment:
```suggestion
                    <field name="total_amount" optional="hide" decoration-bf="1" groups="base.group_multi_currency"/>
```
those values are already retrieved automatically from the field

Reply:
somehow it is needed, I think because the currency can change..